{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2016-2515",
      "ASSIGNER": "secalert@redhat.com"
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-399"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://github.com/hueniverse/hawk/issues/168",
          "name": "https://github.com/hueniverse/hawk/issues/168",
          "refsource": "CONFIRM",
          "tags": [
            "Patch"
          ]
        },
        {
          "url": "https://github.com/hueniverse/hawk/commit/0833f99ba64558525995a7e21d4093da1f3e15fa",
          "name": "https://github.com/hueniverse/hawk/commit/0833f99ba64558525995a7e21d4093da1f3e15fa",
          "refsource": "CONFIRM",
          "tags": [
            "Patch"
          ]
        },
        {
          "url": "http://www.openwall.com/lists/oss-security/2016/02/20/2",
          "name": "[oss-security] 20160220 Re: CVE for nodejs hawk",
          "refsource": "MLIST",
          "tags": []
        },
        {
          "url": "https://nodesecurity.io/advisories/77",
          "name": "https://nodesecurity.io/advisories/77",
          "refsource": "MISC",
          "tags": []
        },
        {
          "url": "http://www.openwall.com/lists/oss-security/2016/02/20/1",
          "name": "[oss-security] 20160219 CVE for nodejs hawk",
          "refsource": "MLIST",
          "tags": []
        },
        {
          "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1309721",
          "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1309721",
          "refsource": "CONFIRM",
          "tags": []
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "Hawk before 3.1.3 and 4.x before 4.1.1 allow remote attackers to cause a denial of service (CPU consumption or partial outage) via a long (1) header or (2) URI that is matched against an improper regular expression."
        }
      ]
    }
  },
  "configurations": {
    "nodes": [
      {
        "operator": "OR",
        "cpe_match": [
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:a:hawk_project:hawk:4.1.0:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:a:hawk_project:hawk:3.1.2:*:*:*:*:*:*:*"
          }
        ]
      }
    ]
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "vectorString": "AV:N/AC:L/Au:N/C:N/I:N/A:C",
        "accessVector": "NETWORK",
        "accessComplexity": "LOW",
        "authentication": "NONE",
        "confidentialityImpact": "NONE",
        "integrityImpact": "NONE",
        "availabilityImpact": "COMPLETE",
        "baseScore": 7.8
      },
      "severity": "HIGH",
      "exploitabilityScore": 10,
      "impactScore": 6.9,
      "obtainAllPrivilege": false,
      "obtainUserPrivilege": false,
      "obtainOtherPrivilege": false,
      "userInteractionRequired": false
    }
  },
  "publishedDate": "2016-04-13T16:59Z",
  "lastModifiedDate": "2016-04-20T15:25Z"
}